roles/default/kiali-deploy/vars/main.yml

```
@@ -38,6 +38,13 @@ kiali_vars:
     {{ kiali_defaults.auth }}
     {%- endif -%}
 
+  clustering: |
+    {%- if clustering is defined and clustering is iterable -%}
+    {{ kiali_defaults.clustering | combine((clustering| stripnone), recursive=True) }}
```

super-trivial but triggered my OCD.. put space before |

```suggestion
    {{ kiali_defaults.clustering | combine((clustering | stripnone), recursive=True) }}
```